bin/node/cli/Cargo.toml

```diff
@@ -40,84 +42,84 @@ structopt = { version = "0.3.8", optional = true }
-frame-system = { version = "2.0.0", path = "../../../frame/system" }
-pallet-balances = { version = "2.0.0", path = "../../../frame/balances" }
-pallet-transaction-payment = { version = "2.0.0", path = "../../../frame/transaction-payment" }
-frame-support = { version = "2.0.0", default-features = false, path = "../../../frame/support" }
-pallet-im-online = { version = "2.0.0", default-features = false, path = "../../../frame/im-online" }
-pallet-authority-discovery = { version = "2.0.0",  path = "../../../frame/authority-discovery" }
+pallet-indices = { version = "2.0.0-dev", path = "../../../frame/indices" }
+pallet-timestamp = { version = "2.0.0-dev", default-features = false, path = "../../../frame/timestamp" }
+pallet-contracts = { version = "2.0.0-dev", path = "../../../frame/contracts" }
+frame-system = { version = "2.0.0-dev", path = "../../../frame/system" }
+pallet-balances = { version = "2.0.0-dev", path = "../../../frame/balances" }
+pallet-transaction-payment = { version = "2.0.0-dev", path = "../../../frame/transaction-payment" }
+frame-support = { version = "2.0.0-dev", default-features = false, path = "../../../frame/support" }
+pallet-im-online = { version = "2.0.0-dev", default-features = false, path = "../../../frame/im-online" }
+pallet-authority-discovery = { version = "2.0.0-dev",  path = "../../../frame/authority-discovery" }
 
 # node-specific dependencies
-node-runtime = { version = "2.0.0", path = "../runtime" }
-node-rpc = { version = "2.0.0", path = "../rpc" }
-node-primitives = { version = "2.0.0", path = "../primitives" }
-node-executor = { version = "2.0.0", path = "../executor" }
+node-runtime = { version = "2.0.0-dev", path = "../runtime" }
+node-rpc = { version = "2.0.0-dev", path = "../rpc" }
+node-primitives = { version = "2.0.0-dev", path = "../primitives" }
+node-executor = { version = "2.0.0-dev", path = "../executor" }
 
 # CLI-specific dependencies
-sc-cli = { version = "0.8.0", optional = true, path = "../../../client/cli" }
-frame-benchmarking-cli = { version = "2.0.0", optional = true, path = "../../../utils/frame/benchmarking-cli" }
-node-transaction-factory = { version = "0.8.0", optional = true, path = "../transaction-factory" }
-node-inspect = { version = "0.8.0", optional = true, path = "../inspect" }
+sc-cli = { version = "0.8.0-dev", optional = true, path = "../../../client/cli" }
+frame-benchmarking-cli = { version = "2.0.0-dev", optional = true, path = "../../../utils/frame/benchmarking-cli" }
+node-transaction-factory = { version = "0.8.0-dev", optional = true, path = "../transaction-factory" }
+node-inspect = { version = "0.8.0-dev", optional = true, path = "../inspect" }
 
 # WASM-specific dependencies
 wasm-bindgen = { version = "0.2.57", optional = true }
 wasm-bindgen-futures = { version = "0.4.7", optional = true }
-browser-utils = { path = "../../../utils/browser", optional = true }
+browser-utils = { path = "../../../utils/browser", optional = true , version = "0.8.0-dev"}
```

Comment:
for cases like this, cargo unleash will impose a CI test in the future so that will never get merged!